Plot.R

```diff
@@ -113,241 +134,659 @@ parsesubject <- function(row, half){
 
+TimeList <- c(TimeList_window1, TimeList_window2)
 ###############################
 # Heatmap of variance explained
 ###############################
 
-transformed.communication.activity <- readr::read_csv("transformed.communication.activity.csv")
+transformed.communication.activity <- 
+  readr::read_csv("transformed.communication.activity.csv")
 transformed.communication.activity["X1"] = NULL
 
 # Two 48 Hour visits containing measurements of blood pressure, heart rate
 BP_HR <- readr::read_csv("heartrate.bp.csv")
-Full <- dplyr::full_join(BP_HR, transformed.communication.activity, by="TimeSubjectIndex")
-bp.HR.com.act <- na.omit(Full)
+act.com.bp.hr.vars4months <- dplyr::full_join(BP_HR, transformed.communication.activity,
+                         by = "TimeSubjectIndex")
+bp.HR.com.act <- na.omit(act.com.bp.hr.vars4months)
 postscript("Variability.Act.Com.BP.eps")
 PlotR2s(bp.HR.com.act[,5:dim(bp.HR.com.act)[2]], 
-        "Variance Explained in Activity, Communication, Biometric Data (Visits 1 and 2)", "BP")
+        paste0("Variance Explained in Activity,", 
+              "Communication, Biometric Data (Visits 1 and 2)"), "BP")
+
 dev.off()
 
 # 48 hour visit 1
 Visit1_bp.HR.com.act <- subset(bp.HR.com.act, Times %in% TimeList_window1)
 
-postscript("Varability.Act.Com.BP.Visit1")
+postscript("Varability.Act.Com.BP.Visit1.Feb.eps")
 PlotR2s(Visit1_bp.HR.com.act[, 5:dim(Visit1_bp.HR.com.act)[2]],
-        "Variance Explained in Activity, Communication, Biometric Data(Visit 1)", "BP")
+        " Variance Explained in Activity, Communication, Biometric Data(Visit 1)",
+        "BP")
 dev.off()
 
 # 48 hour visit 2
 Visit2_bp.HR.com.act <- subset(bp.HR.com.act, Times %in% TimeList_window2)
-postscript("Varability.Act.Com.BP.Visit2")
+postscript("Varability.Act.Com.BP.Visit2.Feb.eps")
 PlotR2s(Visit2_bp.HR.com.act[, 5:dim(Visit2_bp.HR.com.act)[2]],
         "Variance Explained in Activity, Communication, Biometric Data (Visit 2)", "BP")
 dev.off()
 
+energy <- read.csv("energy.csv")
+energy["X"] <- NULL
+energy["Times"] <- apply(energy, 1, ParseSubject, 1)
+
+Energy.4months <- dplyr::full_join(transformed.communication.activity, energy,
+                                   by = "TimeSubjectIndex")
+Energy.4months <- na.omit(Energy.4months)
+
+Full.with.energy <- dplyr::full_join(bp.HR.com.act, energy,
+                                     by = "TimeSubjectIndex")
+Full.with.energy <- na.omit(Full.with.energy)
+Full.with.energy["Subject"] <- NULL
+
+postscript("Variance_Explained_WithEnergy.eps")
+# Plot R^2s excluding Times variable
+PlotR2s(Full.with.energy[, 6:dim(Full.with.energy)[2] - 1],
+        paste0("Variance Explained in Activity, Communication, Blood Pressure,",
+              " and Energy Variables"),
+        "4monthenergy")
+dev.off()
+
+# All four months by subject (no blood pressure/heart rate/ ActCom data)
+transformed.communication.activity["Subject"] <- (apply(transformed.communication.activity, 1, ParseSubject, 2))
```

Comment:
This line looks like it is over 80 char and I think you have an extra set of parentheses